src/features/contacts/view/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button, Card, CardContent, Grid, TextField } from "@mui/material";
```

Comment:
why not make readonly mode for edit control instead of making one more control with the same components for viewing entity?